src/stores/map/mapSlice.ts

```diff
@@ -1,45 +1,48 @@
-import { createSlice, PayloadAction } from '@reduxjs/toolkit'
-// import { getMapThunk } from './mapThunk'
-import { MapData, MapState } from './mapType'
-
-const initialState: MapState = {
-  data: [],
-  loading: false,
-  error: false,
-}
-
-const mapSlice = createSlice({
-  name: 'map',
-  initialState,
-  reducers: {
-    getData: (state, action: PayloadAction<MapData[]>) => {
-      state.data = action.payload
-      // console.log(action.payload)
-    },
-    removeData: (state) => {
-      state.data = []
-    },
-  },
-  extraReducers(builder) {
-    // builder.addCase(getMapThunk.pending, (state) => {
-    //   state.loading = true
-    //   state.error = false
-    // })
-    // builder.addCase(
-    //   getMapThunk.fulfilled,
-    //   (state, action: PayloadAction<MapData[]>) => {
-    //     state.data = action.payload
-    //     state.loading = false
-    //     state.error = false
-    //   }
-    // )
-    // builder.addCase(getMapThunk.rejected, (state) => {
-    //   state.loading = false
-    //   state.error = true
-    // })
-  },
-})
-
-// export { getMapThunk }
-export const { getData, removeData } = mapSlice.actions
-export default mapSlice.reducer
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+// import { getMapThunk } from './mapThunk'
+import { MapState } from './mapType'
+
+const initialState: MapState = {
+  data: [],
+  loading: false,
+  error: false,
+}
+
+const mapSlice = createSlice({
+  name: 'map',
+  initialState,
+  reducers: {
+    getData: (
+      state,
+      action: PayloadAction<kakao.maps.services.PlacesSearchResultItem[]>

```

Comment:
(p3) 타입이 불필요하게 긴 것 같은데요, PlacesSearchResultItem 타입을 import 하는 방법이 없을지 점검 부탁드려요~
```suggestion
      action: PayloadAction<PlacesSearchResultItem[]>
```